.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug Report 🐞
+about: Something isn't working as expected? Here is the right place to report.
+title: '[Bug] <title>'
+labels: Bug, Needs Triage
+assignees: ''
+---
+
+<!--
+  To make it easier for us to help you — please follow the suggested format below.
+
+  Before opening a new issue, please search existing issues.
+-->
+
+# Bug Report
+
+## Relevant information
+
+<!-- Provide as much useful information as you can -->
+
+### Your Environment
+
+- Browser:
+- Browser version:
+- Cypress version:
+- nextjs-auth0 version:
+- cypress-nextjs-auth0 version:
+
+#### Steps to reproduce
+
+1. Step 1
+2. Step 2
+3. Step 3
+
+#### Observed Results
+
+<!-- What happened? This could be a description, log output, etc. -->
+
+#### Expected Results
+
+<!-- What did you expect to happen? -->
+
+#### Relevant Code (optional)
+
+```js
+// TODO(you): code here to reproduce the problem
```

Comment:
Is this code from this repo or code from the consuming app that you'd like to have shared here?

Reply:
This can be a test example from their code, can be a link to our repo with the code causing an issue. Whatever helps reproducing the bug or helping others.